deno/payloads/v10/auditLog.ts

```diff
@@ -826,6 +830,33 @@ export type APIAuditLogChangeKeyExemptRoles = AuditLogChangeData<'exempt_roles',
  */
 export type APIAuditLogChangeKeyExemptChannels = AuditLogChangeData<'exempt_channels', Snowflake[]>;
 
+/**
+ * Returned when a guild forum's available tags gets changed
+ */
+export type APIAuditLogChangeKeyAvailableTags = AuditLogChangeData<'available_tags', APIGuildForumTag[]>;
+
+/**
+ * Returned when a guild forum's default reaction emoji gets changed
+ */
+export type APIAuditLogChangeKeyDefaultReactionEmoji = AuditLogChangeData<
+	'default_reaction_emoji',
+	APIGuildForumDefaultReactionEmoji
+>;
+
+/**
+ * Returned when a channel flag gets changed
+ */
+export type APIAuditLogChangeKeyFlags = AuditLogChangeData<'flags', number>;
+
+/**
```

Comment:
This is a bit of a copy paste fail, it relates to making threads, not sending messages

Reply:
Ah okay, how should we even name this. It only comes when changing this option on a guild forum.
![image](https://github.com/discordjs/discord-api-types/assets/20678496/fb5a4ed2-74e5-4c38-9892-5bbbe4139018)

Reply:
I am pretty sure @vladfrangu is talking about the `APIAuditLogChangeKeyDefaultThreadRateLimitPerUser` below. The description doesn't make sense.